homeassistant/components/sensor/const.py

```diff
@@ -416,6 +416,11 @@ class SensorDeviceClass(StrEnum):
     """
 
 
+IMPLICTIT_NUMERIC_DEVICE_CLASSES = {
```

Comment:
```suggestion
IMPLICIT_NUMERIC_DEVICE_CLASSES = {
```

Reply:
It may be more efficient to refactor the current code btw, but for now I focused on performance.